<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-39NIES78" failures="7" tests="46" name="runners.InvalidScenarioRunner" time="16.198" errors="0" timestamp="2025-02-09T20:24:40 EST" skipped="0">
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.299"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.374"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.301"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.292"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.277"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.295"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.645"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.278">
    <failure message="expected [500] but found [400]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [500] but found [400]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1439)
at stepdefinitions.PostRequestStep.the_response_status_code_should_be_not_found(PostRequestStep.java:65)
at ✽.The response status code should be 400(file:///C:/Users/SMILE/selenium_course/RestAssuredAutomation/src/test/resources/features/postRequest.feature:51)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.284"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.304"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.301"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.404"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.273"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.303">
    <failure message="expected [500] but found [201]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [500] but found [201]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1439)
at stepdefinitions.PostRequestStep.the_response_status_code_should_be_a_response_body_should_have_auto_generated_user_id_address_id_creation_time_and_last_mod_time_field(PostRequestStep.java:111)
at ✽.The response status code should be 201. A response body should have auto-generated user id, address id , creation_time and last_mod_time field.(file:///C:/Users/SMILE/selenium_course/RestAssuredAutomation/src/test/resources/features/postRequest.feature:31)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.307"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.286"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.289"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.371"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.290"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.336"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.287"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.280"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.284"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.312"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.331">
    <failure message="expected [500] but found [201]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [500] but found [201]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1439)
at stepdefinitions.PostRequestStep.the_response_status_code_should_be_a_response_body_should_have_auto_generated_user_id_address_id_creation_time_and_last_mod_time_field(PostRequestStep.java:111)
at ✽.The response status code should be 201. A response body should have auto-generated user id, address id , creation_time and last_mod_time field.(file:///C:/Users/SMILE/selenium_course/RestAssuredAutomation/src/test/resources/features/postRequest.feature:31)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.308">
    <failure message="expected [500] but found [400]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [500] but found [400]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1439)
at stepdefinitions.PostRequestStep.the_response_status_code_should_be_not_found(PostRequestStep.java:65)
at ✽.The response status code should be 400(file:///C:/Users/SMILE/selenium_course/RestAssuredAutomation/src/test/resources/features/postRequest.feature:81)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.284"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.296"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.332"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.327"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.298"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.298">
    <failure message="expected [500] but found [400]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [500] but found [400]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1439)
at stepdefinitions.PostRequestStep.the_response_status_code_should_be_not_found(PostRequestStep.java:65)
at ✽.The response status code should be 400(file:///C:/Users/SMILE/selenium_course/RestAssuredAutomation/src/test/resources/features/postRequest.feature:51)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.277"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.299">
    <failure message="expected [500] but found [400]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [500] but found [400]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1439)
at stepdefinitions.PostRequestStep.the_response_status_code_should_be_not_found(PostRequestStep.java:65)
at ✽.The response status code should be 400(file:///C:/Users/SMILE/selenium_course/RestAssuredAutomation/src/test/resources/features/postRequest.feature:71)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.323"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.278"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.279"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.939"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.269">
    <failure message="expected [500] but found [400]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [500] but found [400]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1439)
at stepdefinitions.PostRequestStep.the_response_status_code_should_be_not_found(PostRequestStep.java:65)
at ✽.The response status code should be 400(file:///C:/Users/SMILE/selenium_course/RestAssuredAutomation/src/test/resources/features/postRequest.feature:71)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.285"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.288"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.379"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="1.532"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.276"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.294"/>
  <testcase classname="runners.InvalidScenarioRunner" name="runScenario" time="0.334"/>
</testsuite> <!-- runners.InvalidScenarioRunner -->
